### ISSUE TITLE
NEW Classification Datasets: `imagenet10`, `imagenet100`, `imagenet1000`

### ISSUE THREAD

**Issue by glenn-jocher** [ultralytics/ultralytics#1071]
All, FYI I've created 3 new classification datasets for use with debugging/tests/benchmarks: `imagenet10`, `imagenet100`, `imagenet1000`.

These are super small versions of imagenet that train/val in seconds with only 1 image per class (with all 1000 classes, only 100 classes, and only 10 classes). They are reduced in size to `imgsz=160` and compressed with PIL so that i.e. imagenet10 is only 70kB, imagenet1000 is only 7MB.

### CI 

I've migrated all of our CI, tests and benchmarks to use them, i.e. benchmarks are now on imagenet100 (top5 accuracy is 0.71 for YOLOv8n-cls on imagenet100). 
<img width="635" alt="Screenshot 2023-02-20 at 14 56 41" src="https://user-images.githubusercontent.com/26833433/220128254-cee45a94-83b5-41a5-9ade-6cc77681e3bc.png">

### Download

You can download them here:
https://github.com/ultralytics/yolov5/releases/download/v1.0/imagenet10.zip
https://github.com/ultralytics/yolov5/releases/download/v1.0/imagenet100.zip
https://github.com/ultralytics/yolov5/releases/download/v1.0/imagenet1000.zip

### Usage

```bash
yolo train model=yolov8n-cls.pt data=imagenet100
```
or
```python
from ultralytics import YOLO

model = YOLO('yolov8n-cls.pt')
results = model.train(data='imagenet100', imgsz=160)
```

**Comment by osamaalschame** [comment]
if i have my own dataset how the data file will looks like?

**Comment by liujia761** [comment]
Hello, can you give a set of training imagenet classification parameters and configuration, my training accuracy and authority can not match？

**Comment by pderrenger** [comment]
@liujia761 certainly! You can find detailed information on training and configuration parameters for imagenet classification in the YOLOv8 documentation. Check out the [training configuration](https://docs.ultralytics.com/tasks/classify#training-parameters) section for a comprehensive guide on setting up the parameters and achieving optimal training accuracy. If you have specific questions regarding the parameters or need further assistance, feel free to ask!

**Comment by sjiang95** [comment]
Hello,

may I ask what are those "copy"s?

```shell
MINGW64 ~/Downloads/imagenet1000 $ ls train/n01440764
'n01440764_2923 copy 2.JPEG'  'n01440764_2923 copy.JPEG'   n01440764_2923.JPEG
```

Their checksums are identical.

```shell
$ sha256sum.exe train/n01440764/n01440764_2923\ copy\ 2.JPEG
e89bc6b45c8783d410646f34e79c1731f3c779acd89965bf1580fe95c69efa79 *train/n01440764/n01440764_2923 copy 2.JPEG
$ sha256sum.exe train/n01440764/n01440764_2923\ copy.JPEG
e89bc6b45c8783d410646f34e79c1731f3c779acd89965bf1580fe95c69efa79 *train/n01440764/n01440764_2923 copy.JPEG
$ sha256sum.exe train/n01440764/n01440764_2923.JPEG
e89bc6b45c8783d410646f34e79c1731f3c779acd89965bf1580fe95c69efa79 *train/n01440764/n01440764_2923.JPEG
```

They exist in all the presented `10`, `100` and `1000` datasets.

**Comment by Rahulbiradar9** [comment]
Hello @glenn-jocher,

Since the `imagenet10`, `imagenet100`, and `imagenet1000` datasets are fully integrated, used across the repository's test/validation pipelines, and listed under the supported classification datasets in the documentation, it looks like this announcement issue can be closed. 

Thanks!
